pkgs/development/python-modules/twisted/default.nix

```diff
@@ -27,7 +41,14 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ zope_interface incremental automat constantly hyperlink pyhamcrest attrs setuptools typing-extensions ];
 
-  passthru.extras.tls = [ pyopenssl service-identity idna ];
+  passthru.extras-require = rec {
+    tls = [ pyopenssl service-identity idna ];
+    conch = [ pyasn1 cryptography appdirs bcrypt ];
+    conch_nacl = conch ++ [ pynacl ];
+    serial = [ pyserial ];
+    http2 = [ h2 priority ];
+    contextvars = lib.optionals (pythonOlder "3.7") [ contextvars ];
+  };
```

Comment:
I am currently doing major changes in twisted. This needs to be adjusted after that or I cherry-pick those changes into my PR.

Probably doing the later.

Reply:
Why not merge this and you rebase?

Reply:
Because my PR needs to target staging because it triggers a rebuild and I want to avoid that someone else needs to resolve the more complicated merge conflict.

Reply:
So you'll have to wait a few hours before merging, that's fine.